### DISCUSSION TITLE
password root

### DISCUSSION THREAD

**Discussion by fabio8v** [minios-linux/minios-live#15] [Q&A] [answered]
hi
what is the password of the root?

**Comment by AndreAlatus** [comment]
Hi, I needed to. I did not find, but ... sudo ->  Enter -> passwd root = put your password for root

**Comment by michaelcton** [comment]
There is no default password for root—if you enter `sudo passwd` on the command line it will ask you to create a new one without asking for current credentials. Keep in mind the user logged into the session by default is "live".

**Comment by crim50n** [comment, marked as answer]
```
live:evil
root:toor
```
Changing the password using the classic method for all systems (`sudo passwd`) will not help, because the password will be overridden by the `minios.conf` configuration file, which is located in the `minios` folder on the flash drive.
